modules/admin_manual/pages/enterprise/collaboration/collabora_secure_view.adoc

```diff
@@ -1,11 +1,12 @@
-= Secure View
+= Collabora Online / Secure View
 :toc: right
 :secure-view-label: Secure View (with watermarks)
-:richdocuments-url: https://marketplace.owncloud.com/apps/richdocuments
 :page-aliases: collabora_online_integration.adoc
 
 == Introduction
 
+Collabora Online allows you to work with all kinds of Collabora office documents directly in your browser. This application can connect to a Collabora Online (or other) server (WOPI-like Client). ownCloud is the WOPI Host.
+
 When Collabora Online is properly setup and integrated into ownCloud Server, Secure View functionality is available. Secure View is a mode where users can place limitations on files and folders that are shared.
```

Comment:
```suggestion
When Collabora Online is properly setup and integrated into ownCloud Server, secure view functionality is available. Secure view is a mode where users can place limitations on files and folders that are shared.
```

Reply:
Secure View is name of product I think.

Reply:
Ah, okay, then it should be capitalized.